### PR TITLE
[AN-4845] Missing info in connect system msg

### PR DESCRIPTION
**Pull request by hakonbo**
#### APK
[Download build #8260](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8260/artifact/build/artifact/wire-dev-PR469-8260.apk)
[Download build #8264](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8264/artifact/build/artifact/wire-dev-PR469-8264.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/common/views/UserDetailsView.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import com.waz.zclient.utils.StringUtils
+import com.waz.zclient.{R, ViewHelper}
+
+class UserDetailsView(val context: Context, val attrs: AttributeSet, val defStyle: Int) extends LinearLayout(context, attrs, defStyle) with ViewHelper {
+  def this(context: Context, attrs: AttributeSet) = this(context, attrs, 0)
+  def this(context: Context) = this(context, null)
+
+  private lazy val userNameTextView: TextView = findById(R.id.ttv__user_details__user_name)
+  private lazy val userInfoTextView: TextView = findById(R.id.ttv__user_details__user_info)
+  LayoutInflater.from(context).inflate(R.layout.user__details, this, true)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/common/views/UserDetailsView.scala]: I think `ViewHelper` has the method `inflate` already there as a shortcut :)

File: app/src/main/scala/com/waz/zclient/common/views/UserDetailsView.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  val handleText = handle.map {
+    case Some(handle) => StringUtils.formatUsername(handle.string)
+    case None => ""
+  }
+
+  val contactText = for {
+    c <- firstContact
+    n <- displayName
+  } yield {
+    if (c.isDefined) {
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/common/views/UserDetailsView.scala]: You can also do:

```
c match {
 case Some(cont) if cont.name == n => //
 case Some(cont) => //
 case _ => ""
}
```

**Comment by red-avtovo** [comment]
Nothing happened. YOU added YOU still exist

**Comment by deanrobertcook** [comment]
build again

**Pull request merged**